Source/Controls/CommandBar/for_CommandBar/when_initiating_a_command_bar.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+import 'aurelia-polyfills';
+import { CommandBar } from '../CommandBar';
+import { commandLocation } from '../commandLocation';
+import { commandLabelPosition } from '../commandLabelPosition';
+
+describe('when initiating a command_bar without any properties set', () => {
+  let commandBar: CommandBar;
+
+  beforeEach(() => {
+    commandBar = new CommandBar();
+  });
+
+  it('should have commandLocation set to "right"', () => commandBar.commandLocation.should.equal(commandLocation.right));
+  it('should have commandLabelPostition set to "right"', () => commandBar.commandLabelPosition.should.equal(commandLabelPosition.right));
+  it('should have zIndex set to 99', () => commandBar.zIndex.should.equal(99));
+  it('should not display secondary commands', () => commandBar.displaySecondaryCommands.should.be.false);
+});
+
+describe('when initiating a command_bar with commandLocation set', () => {
```

Comment:
You should split these into multiple files and instead have a `when_initiating` folder with the different specifications as files underneath it. I would also use initializing or creating as the verb and possibly leave out *command_bar* in the spec name, as it is implicitly sitting in the context of the command_bar. 

Have a look [here](https://github.com/dolittle-interaction/JavaScript.Client/tree/master/Source/commands/for_CommandCoordinator) for an example of how we tend to do things. Unfortunately we have yet to document our current expectations around structure and how specs are to be structured and written.

But a rule of thumb is if you have multiple things within a «verb» (action), then create a folder and have the files start with the state you’re specifying or even start with an «and_» or similar. For your scenario the files would be «without_any_properties_set.ts», «with_command_label_position_set».